code/__defines/inventory_sizes.dm

```diff
@@ -1,5 +1,5 @@
 // The below should be used to define an item's w_class variable.
-// Example: w_class = ITENSIZE_LARGE
+// Example: w_class = ITEN_SIZE_LARGE
```

Comment:
correcting a typo in a comment doesn't do anything, but this is still wrong, theres no N in ITEM

Reply:
Lol. My bad, my eyes saw the lack of underscore but not that the letter was wrong. I'll get it.